### PR TITLE
Weekly Roundup - 14SEP2018

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description

The goal of this PR is to update the weekly roundup email per @vs77bb's provided copy changes.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

emails, roundup

##### Testing

Locally

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -533,14 +533,15 @@ def render_new_bounty_roundup(to_email):
 Hi there,
 </p>
 <p>
-We're excited to formally announce an expansion of our ECF Partnership, bringing $60K in funding to the <a href="https://medium.com/@mitchkosowski/cbe1d2d1c071">Ethereum Infrastructure layer</a>!
```
Comment (owocki) [inline app/retail/emails.py]: subject line needs changed

**Comment by owocki** [comment]
i get a `django.urls.exceptions.NoReverseMatch: Reverse for 'leaderboard' with no arguments not found. 1 pattern(s) tried: ['leaderboard/(.*)']

` when i try to preview this in the web browser

File: app/retail/emails.py (hunk 1 above)
Comment (mbeacom) [reply app/retail/emails.py]: https://github.com/gitcoinco/web/commit/8445c4408b1fa3a7edf305810e3f049ee78faa63

File: app/retail/emails.py [hunk 2]
```diff
@@ -533,14 +533,15 @@ def render_new_bounty_roundup(to_email):
...
-As stated in the article, if you know of any open source projects / specific Github issues which could use some funding, <a href="https://gitcoin.co/requests">make a Gitcoin Request</a> and comment 'ECF'.
-We're very excited to make it easy to contribute to the ecosystem, while getting paid.
+Hacktoberfest is a month long celebration of open source. As Gitcoiners, we support anything that grows open source. Next Monday, we're going to formally
+announce our plans for Hacktoberfest...but you can see them <a href="https://medium.com/p/98825f199af2/edit">here today</a>!
```
Comment (owocki) [inline app/retail/emails.py]: edit link, should be a preview link @vs77bb

**Comment by owocki** [comment]
testing link for next time => http://localhost:8000/_administration/email/roundup

**Comment by mbeacom** [comment]
I intended on checking it out before I starting rolling out the email.  Apologies...  In any case, that was an issue I introduced on the template refactor.  Meant to use `_leaderboard`.
Thanks!

File: app/retail/emails.py (hunk 2 above)
Comment (vs77bb) [reply app/retail/emails.py]: @owocki in practice this doesn't change anything for readers...right? I will be careful to use preview links if so

**Pull request merged**